packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -373,4 +373,20 @@ describe('ComboBox', () => {
     buttonElement.simulate('click');
     expect(returnUndefined.mock.calls.length).toBe(1);
   });
+
+  it('merges callout classNames', () => {
+    ReactTestUtils.renderIntoDocument<ComboBox>(
+      <ComboBox
+        options={ DEFAULT_OPTIONS }
+        calloutProps={ { className: 'foo' } }
+      />
+    );
+
+    setTimeout(() => {

```

Comment:
This setTimeout is needed due to the calloutProps not being fully propagated on first UI tick. Please let me know if there is a better solution to solve that problem.